### PR TITLE
[Feature] Reload function for listview and mapview without page refresh.

### PR DESCRIPTION
**Pull request by Pawan96**

### CODE REVIEW

File: public/html/listView.html [hunk 1]
```diff
@@ -34,12 +34,21 @@
...
+    
 
     <!-- Location list -->
     <ul class='collection'></ul>
 
+    <!-- List Refresh Button -->
+    <div>
+        <a id="btnListRefresh" class="waves-effect waves-light btn blue"><i class="material-icons left">refresh</i>Refresh List</a>
+    </div>
+
```
Comment (erickao5858) [inline public/html/listView.html:51]: Remove all the empty lines in this file.
And it is better that the button has no text but a refresh icon -> a refresh icon is enough for that button.
Also, can you make the button right above the menu button?

File: public/html/mapView.html [hunk 2]
```diff
@@ -38,7 +38,13 @@
     <div class='fixed-action-btn'>
         <a href='#' class='btn-floating btn-large waves-effect waves-light blue'><i class='medium material-icons'>menu</i></a>
     </div>
+    <!-- Map Refresh Button -->
+    <div>
+        <a id="btnMapRefresh" class="waves-effect waves-light btn blue"><i class="material-icons left">refresh</i>Refresh Map</a>
+    </div>
+
     <div id='map'></div>
+
```
Comment (erickao5858) [inline public/html/mapView.html:47]: Same to listview.html

**Review by erickao5858 — COMMENTED** [review]
I think it is enough for a close. Just change some detail of the button and I will merge it.

File: public/html/listView.html (hunk 1 above)
Comment (Pawan96) [reply public/html/listView.html:51]: image

**Pull request closed without merging**